### PR TITLE
Add more documentation to the wheel key module

### LINKED ISSUE #35683: Salt wheel key documentation improvements
**Background:** I was interested in coding up accepting minion keys. I found https://docs.saltstack.com/en/latest/ref/wheel/all/salt.wheel.key.html, but I wanted to know if my `accept` went through or not (since there was a risk my key was rejected). I looked at https://github.com/saltstack/salt/blob/develop/salt/wheel/key.py#L49 and discovered that it actually returns a result of some kind. However, the PyDoc doesn't document that.

**Proposal:** To document the return results on the functions in https://github.com/saltstack/salt/blob/develop/salt/wheel/key.py.

### PR DESCRIPTION
**Pull request by rallytime**
### What does this PR do?

The documentation for `salt/wheel/key.py` was lacking. This updates the documentation for args/kwargs, return structures, and gives some examples of how to use the functions in python.
### What issues does this PR fix or reference?

Fixes saltstack/salt#35683
### Tests written?

No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

### CODE REVIEW

File: salt/wheel/key.py [hunk 1]
```diff
@@ -1,6 +1,29 @@
 # -*- coding: utf-8 -*-
 '''
-Wheel system wrapper for key system
+Wheel system wrapper for key system.
```
Comment (JensRantil) [inline salt/wheel/key.py]: _the_ key system

Comment (JensRantil) [inline salt/wheel/key.py]: Also, this sentence could probably be improved. It doesn't really say very much to an en user.

File: salt/wheel/key.py [hunk 2]
```diff
@@ -19,101 +42,264 @@
...
+        'priv': '-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEA42Kf+w9XeZWgguzv
+        Iqm6DZvUSuIOjgesoAaK0waBnKWSbagO\nYbAHs/mCYD4xuGTVk/Nt4w8JpRCJRhLKnQz+4l9Z
+        E28swE+TOzK4rXVqGo4sAQ70\nWkG759e/pWpaJi9LlTAS9w0FaJLC53nrCii4AXxldlHU9jub
+        b4/AbGp6FKz6C53R\nF6VZcaL6GZUF/mjWkPODGUu2bEs8FDXSHFUwLajoSOUPwSgaxW2ggTyy
+        nm0XgELM\nzkCmFihv0Ve4c+XPuOdG55Jp8B5oG2xAtviUaEEBBPfamX9gGPQTpN9e8HwcZjXQ
+        \nnmg8OrcUl10WHw09SDWLOlnW+ueTWugEQpPtiQIDAQABAoIBAFPIf1IusPin3WDg\nlKydBW
+        UGSg7dhKdoxH6ianyfJRrnFzE4fzQXfH8gLJ3xzLg4z0mYpKDJZbBgi3Q6\ncFW1+CTJJTU3/b
+        FOoN1rLwka/mGSiTxKmla3adlfccotidIQEzWBZTuKnLePfCl2\njthLcKF7iJc0R9gC1p8gxL
+        It+ZWsBQyBH3+WHeWJ/sI9YWDpYKxmGNdcCIFH2v+L\n+V8y1R1JUsUbcqOQcxKxl7B+mKX9Bn
+        AOhX8pW4xpvpOzaIA2BpNkgmaqY+9/nFE1\nolA8ngZEImRoU5tmMquThneU1ACzRH7ZoceIX+
```
Comment (JensRantil) [inline salt/wheel/key.py]: Maybe the full key isn't necessary here? Could be reduced a bit to "...".

File: salt/wheel/key.py [hunk 3]
```diff
@@ -133,7 +319,65 @@ def gen(id_=None, keysize=2048):
...
+        gpwUTTtIZrTwRhKlFPn2qcxFuF\nc5mqqpECgYEA65ZhY/V9emOYGanA7VDuRJp+249I5B3bvo
+        8C2yq2uR96EhbM6Q6c\nO8tdk3CaA9a2vkanchP+mi+TwX9hAEuuK5GS+XitDxOAXsM9OnHn9b
+        sX5snncrJL\naMplOonzjZGg6/tKDJl/SErr7hT0ylUtx3X3fqdrJGdKdq67X5ogMPcCgYEA9x
```
Comment (JensRantil) [inline salt/wheel/key.py]: See previous comment about reducing output a bit.

File: salt/wheel/key.py [hunk 4]
```diff
@@ -133,7 +319,65 @@ def gen(id_=None, keysize=2048):
...
+        BBPfamX9gGPQTpN9e8HwcZjXQnmg8OrcUl10WHw09SDWLOlnW+ueTWugEQpPt\niQIDAQAB\n
+        -----END PUBLIC KEY-----',
+        'priv': '-----BEGIN RSA PRIVATE KEY-----\nMIIEpAIBAAKCAQEA42Kf+w9XeZWgguzv
+        Iqm6DZvUSuIOjgesoAaK0waBnKWSbagO\nYbAHs/mCYD4xuGTVk/Nt4w8JpRCJRhLKnQz+4l9Z
+        E28swE+TOzK4rXVqGo4sAQ70\nWkG759e/pWpaJi9LlTAS9w0FaJLC53nrCii4AXxldlHU9jub
+        b4/AbGp6FKz6C53R\nF6VZcaL6GZUF/mjWkPODGUu2bEs8FDXSHFUwLajoSOUPwSgaxW2ggTyy
+        nm0XgELM\nzkCmFihv0Ve4c+XPuOdG55Jp8B5oG2xAtviUaEEBBPfamX9gGPQTpN9e8HwcZjXQ
+        \nnmg8OrcUl10WHw09SDWLOlnW+ueTWugEQpPtiQIDAQABAoIBAFPIf1IusPin3WDg\nlKydBW
+        UGSg7dhKdoxH6ianyfJRrnFzE4fzQXfH8gLJ3xzLg4z0mYpKDJZbBgi3Q6\ncFW1+CTJJTU3/b
+        FOoN1rLwka/mGSiTxKmla3adlfccotidIQEzWBZTuKnLePfCl2\njthLcKF7iJc0R9gC1p8gxL
```
Comment (JensRantil) [inline salt/wheel/key.py]: See previous comment about reducing output a bit.

File: salt/wheel/key.py [hunk 5]
```diff
@@ -133,7 +319,65 @@ def gen(id_=None, keysize=2048):
...
+        It+ZWsBQyBH3+WHeWJ/sI9YWDpYKxmGNdcCIFH2v+L\n+V8y1R1JUsUbcqOQcxKxl7B+mKX9Bn
+        AOhX8pW4xpvpOzaIA2BpNkgmaqY+9/nFE1\nolA8ngZEImRoU5tmMquThneU1ACzRH7ZoceIX+
+        gpwUTTtIZrTwRhKlFPn2qcxFuF\nc5mqqpECgYEA65ZhY/V9emOYGanA7VDuRJp+249I5B3bvo
```
Comment (JensRantil) [inline salt/wheel/key.py]: See previous comment.

**Comment by JensRantil** [comment]
Excellent on my part! Had some minor comments.

File: salt/wheel/key.py [hunk 6]
```diff
@@ -19,101 +42,264 @@
...
+        Ad/THK99DopbyIWwxOW08Wwp5ATr8c+HdSGVDx1k4HbEi7TGCbYBHBJ+AyYe\nAlepenCwCdDu
+        c6B8uczQBYIpa8dYmJ6rnROrtX8CgYEAw/onLjYrmcJ75u1V9pUa\ngKtc2FzLsCg+grkJETTp
+        J/rNW3roHv/ua7dzMpG7LXJSNtsjtDx50d2+Nns2QD60\ntr203B7NJ/t7EFmt946F/d8TtQNE
+        rB8xqAymRRo5Jw18JpK2OOEABy18MmPwdtiA\n6qpxpkSzVmcuhKg6av8VxGUCgYBRBnTnG7uq
+        ddNiRXDTNl3skyedAIiqB7kSAYYx\nKjbndXsT377nAwiCu82erJIzVsKfRVbiiDyPX8mH55g3
+        03FDsWuFNdp/LLwT9PU2\nZ5qA85FA0p/I58mOEqq6dn+op/vApphCpmKCLz/bCDOZoTAlcZ9p
+        nwRcPpO0AJxA\nNzogmwKBgQCFz8TQ4m41dNvTldAEiqRozsDTrjKOgNePOWHzjEnYCzS/FQv/
+        rsTx\nky3GRz3FTQ02tVa2lUqzfn05bAQtrD2wjKt4xoCdgqKVbCXhJ4RIovpkPSESOrJw\nFl
+        QH3/W74X1+WTBlx4R2KGLYBiH+bCCFEQ/Zvcu4Xp4bIOPtRKozEQ==\n
+        -----END RSA PRIVATE KEY-----'}
```
Comment (cachedout) [inline salt/wheel/key.py]: I do hope this isn't a real key. ;]

Comment (JensRantil) [inline salt/wheel/key.py]: :)

Comment (rallytime) [inline salt/wheel/key.py]: Not a key saved or used anywhere. Just generated and output on the CLI for use as an example.

**Comment by rallytime** [comment]
Thanks for the review @JensRantil! I made all of the changes you suggested.

**Comment by JensRantil** [comment]
Looks good on my behalf.

**Comment by cachedout** [comment]
Thanks for pitching in here, @JensRantil. Always very helpful!

**Pull request merged**